src/context-parser-handlebars.js

```diff
@@ -604,11 +624,69 @@ ContextParserHandlebars.prototype.addFilters = function(stateObj, input) {
                     }
                     return filters;
                 } else if (attributeName === "style") {  // CSS
-                    /* we don't support css parser yet
-                    * we use filter.FILTER_NOT_HANDLE to warn the developers for unsafe output expression,
-                    * and we fall back to default Handlebars escaping filter. IT IS UNSAFE.
-                    */
-                    throw 'Unsafe output expression @ attribute style CSS context';
+                    /* html decode the attributeValue before CSS parsing,
+                       we follow the parsing order of the browser */
+                    len = reCss.length;
+                    for (j=0;j<len;++j) {
+                        attributeValue = attributeValue.replace(reCss[j], cssReplaceChar[j]);
+                    }
+                    attributeValue = attributeValue.replace(/ /g, ''); // ok to remove all space and space has been html decoded.
+                    debug("addFilter:attributeValue:["+attributeValue+"]");
+
+                    /* split the string as per http://www.w3.org/TR/css-style-attr/ with ':' and ';' */
+                    var kv = attributeValue.split(';'); // it will return new array even there is no ';' in the string
+                    var v = kv[kv.length-1].split(':'); // only handling the last element
+
+                    if (v.length && v.length === 2) {
+                        filters = [];
+
+                        var prop = v[0],
+                            expr = v[1];
+                        debug("addFilter:prop:"+prop+",expr:"+expr);
+
+                        /* TODO: we can whitelist the property here */
+                        if (prop !== '') { // && whitelisted
+
+                            /* space has been trimmed above */
+                            if (expr.match(/^url\($/)) {
```

Comment:
you need a regexp that also match url case insensitively, and the html encoded versions of `u` `r` `l`